corehq/apps/reports/standard/sms.py

```diff
@@ -240,21 +232,24 @@ def export_table(self):
             row.append(contact_info[2])
         return result
 
-"""
-Displays all sms for the given domain and date range.
-
-Some projects only want the beginning digits in the phone number and not the entire phone number.
-Since this isn't a common request, the decision was made to not have a field which lets you abbreviate
-the phone number, but rather a settings parameter.
 
-So, to have this report abbreviate the phone number to only the first four digits for a certain domain, add 
-the domain to the list in settings.MESSAGE_LOG_OPTIONS["abbreviated_phone_number_domains"]
-"""
 class MessageLogReport(BaseCommConnectLogReport):
+    """
+    Displays all sms for the given domain and date range.
+
+    Some projects only want the beginning digits in the phone number and not the entire phone number.
+    Since this isn't a common request, the decision was made to not have a field which lets you abbreviate
+    the phone number, but rather a settings parameter.
+
+    So, to have this report abbreviate the phone number to only the first four digits for a certain domain, add
+    the domain to the list in settings.MESSAGE_LOG_OPTIONS["abbreviated_phone_number_domains"]
+    """
     name = ugettext_noop('Message Log')
     slug = 'message_log'
+    ajax_pagination = True
 
     exportable = True
+    exportable_all = True
```

Comment:
When I click the export button now, it gives me a message that the export will be emailed to me. Was that intentional, and can we keep it as a regular download?